tests/units/test_parser.py

```diff
@@ -111,6 +138,19 @@ def test_conversion():
     for source, dest in conversions.items():
         assert convert_units(convert_units(1, source, dest), dest, source) == 1
 
+    # Verify that convert_units respects scaling factors
+    assert -1e-8 < convert_units(100, 'g / 100 mL', 'g/cc') - 1 < 1e-8
+    assert -1e-8 < convert_units(1, "g / 2.5 cm", "g / 25 mm") - 1 < 1e-8
+
+
+def test_get_base_units():
+    """Test that base units & conversions make sense."""
+    from gemd.units.impl import _REGISTRY
+    assert get_base_units("degC") == (_REGISTRY("kelvin"), 1, 273.15)
+    assert get_base_units("degC") == get_base_units(_REGISTRY("degC"))
```

Comment:
If both functions get_base_units() return the wrong value, the assert is still true. The value being compared, should not rely on a method we are asserting against.

Reply:
We're verifying consistent polymorphic behavior, i.e. gives the same result for a string and a Unit object.  Though I did make an oversight here, since it should be `_REGISTRY("degC").u`